client/src/containers/statisticsPage/index.js

```diff
@@ -0,0 +1,68 @@
+import React, { Component } from 'react';
+import { Progress } from 'antd';
+import axios from 'axios';
+import AdminContainer from '../../components/AdminContainer';
+import './style.css';
+
+class Statistics extends Component {
+  state = {
+    data: {
+      cohortsCount: '',
+      projectsCount: '',
+      studentsCount: '',
+    },
+  };
+
+  async componentDidMount() {
+    try {
+      const {
+        data: { data },
+      } = await axios.get('/api/v1/stats');
+      this.setState({ data });
+    } catch (err) {
+      console.log('There is no data');
+    }
+  }
+
+  render() {
+    const { data } = this.state;
+    return (
+      <div>
+        <AdminContainer>
+          <div className="state">
+            <h1>Control Panel Homepage</h1>
+            <div className="titles">
+              <h4>NumberOfCohorts</h4>
+              <p>{data.cohortsCount}</p>
+            </div>
+            <Progress
+              percent={data.cohortsCount}
+              size="small"
+              status="active"
```

Comment:
Statistics only show numbers, we do not need to show a percentage 
```suggestion
   showInfo={false} 
```